content/docs/capabilities/ppl.mdx

```diff
@@ -76,6 +76,39 @@ If `nor` is used, the user will have access if their email address does not end
   </div>
 </details>
 
+### Multiple Operators in an Action
```

Comment:
Sorry, one more terminology nitpick: do the operators belong to the "action", or to the "rule"?

Based on the above sentence "The object represents a rule where the action is the key and the value is an object containing the logical operators" I wonder if it would be more precise to label this more like "Multiple Operators in One Rule"?